From 5bcdb553e5e465f5c4220dfb45c407e3bc18b4af Mon Sep 17 00:00:00 2001
From: Xiaolei Wang <xiaolei.wang@windriver.com>
Date: Tue, 1 Sep 2020 12:51:18 +0800
Subject: [PATCH] pinctrl: freescale: Remove PINCTRL_IMX_SCU dependent option
 IMX_SCU

Fix a compilation warning that PINCTRL_IMX_SCU depends on IMX_SCU. Since
4.19 SDK does not open IMX_SCU, but a separately created IPC interface,
PINCTRL_IMX_SCU does not need to rely on IMX_SCU.

Signed-off-by: Xiaolei Wang <xiaolei.wang@windriver.com>
---
 drivers/pinctrl/freescale/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/pinctrl/freescale/Kconfig b/drivers/pinctrl/freescale/Kconfig
index c7e47843043e..f09f6b89377b 100644
--- a/drivers/pinctrl/freescale/Kconfig
+++ b/drivers/pinctrl/freescale/Kconfig
@@ -8,7 +8,6 @@ config PINCTRL_IMX
 
 config PINCTRL_IMX_SCU
 	bool
-	depends on IMX_SCU
 	select PINCTRL_IMX
 
 config PINCTRL_IMX1_CORE
-- 
2.17.1

